src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -250,7 +250,7 @@ public void setReceiver(String receiver) {
   }
 
   public void initialize() {
-    mentionPattern = Pattern.compile("\\b(" + Pattern.quote(userService.getUsername()) + ")\\b", CASE_INSENSITIVE);
+    mentionPattern = Pattern.compile("\\b(" + Pattern.quote(userService.getUsername()) + ")\\b|[ ]", CASE_INSENSITIVE);
```

Comment:
Btw this doesn’t catch the case where the username is the only thing on the line. Like, this will not match the string "Box-".